src/routes/api/ghost/+server.ts

```
@@ -0,0 +1,169 @@
+        }
+    } catch (error: unknown) {
+        console.error('Ghost API error:', error);
+        const errorMessage = error instanceof Error ? error.message : 'Unknown error occurred';
+        return json({ error: errorMessage }, { status: 500 });
+    }
+};
+
+async function createGhostMember(
+    fetch: typeof globalThis.fetch,
+    email: string,
+    name: string = '',
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)